tests/supervisors/test_watchgodreload.py

```diff
@@ -12,96 +10,196 @@ def run(sockets):
 
 
 def test_watchgodreload(
-    tls_ca_certificate_pem_path, tls_ca_certificate_private_key_path
+    tls_ca_certificate_pem_path, tls_ca_certificate_private_key_path, tmpdir
```

Comment:
could throw another PR to keep things separated but since it's not used here we should drop it
```suggestion
 tmpdir
```